Makefile

```diff
@@ -86,7 +86,7 @@ FV_FELIXIMAGE ?= calico/felix:latest-$(BUILDARCH)
 
 # If building on amd64 omit the arch in the container name.  Fixme!
 ifeq ($(BUILDARCH),amd64)
-        PROTOC_VER?=v0.1
+        PROTOC_VER=v0.1
```

Comment:
This prevents us from using an invocation like `make <something> PROTOC_VER=v0.2` for trying out a new protoc.  I'd prefer if we could keep the `?=`.  Maybe the non-amd64 case could be handled in an `else`?

Reply:
>>  I repushed v0.1 as calico/protoc:v0.1-amd64, does that help?
yes that helps.  I can eliminate "PROTOC_CONTAINER=calico/protoc:$(PROTOC_VER)" from inside the amd ifeq.  The entire ifeq ($(BUILDARCH),amd64) block is a hack to handle the transition to the new image names.  The more I can eliminate from inside the ifeq the better.   

We could define PROTOC_VER ?= v0.1 (and GO_BUILD_VER ?= v0.10) just once outside the amd "if".  The amd build should work,  other arch native builds can override the values to meet the builders needs.

For example:
make <something> ARCH=ppc64le GO_BUILD_VER=<latest> PROTOC_VER=<latest>  

I think this is cleaner and easer to keep updated.  In the future if we push multi-arch manifest for the protoc and go-build images then all arch builds should just work.